src/libs/m4a_2.c

```diff
@@ -85,7 +85,8 @@ void m4aSoundInit(void)
         mplayInfo->memAccArea = gMPlayMemAccArea;
     }
 
-    memcpy(&gPokemonCrySong, &gPokemonCrySongTemplate, sizeof(struct PokemonCrySong));
+    gPokemonCrySong = gPokemonCrySongTemplate;
+//    memcpy(&gPokemonCrySong, &gPokemonCrySongTemplate, sizeof(struct PokemonCrySong));
```

Comment:
Any reason to keep this comment?

Reply:
No, I just forgot to delete it.  I left it in case my change would have broken it